shardingsphere-sql-parser/shardingsphere-sql-parser-binder/src/test/java/org/apache/shardingsphere/sql/parser/binder/metadata/table/SchemaMetaDataLoaderTest.java

```diff
@@ -0,0 +1,107 @@
+package org.apache.shardingsphere.sql.parser.binder.metadata.table;
+
+import org.apache.shardingsphere.sql.parser.binder.metadata.schema.SchemaMetaDataLoader;
+import org.junit.Test;
+
+import javax.sql.DataSource;
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.SQLFeatureNotSupportedException;
+import java.util.LinkedList;
+import java.util.logging.Logger;
+
+public final class SchemaMetaDataLoaderTest {
+    private static int minIdleCount = 3;
+    private static int maxIdleCount = 10;
+    private static int currentIdleCount = 0;
+    private static final LinkedList<Connection> connectionsPool = new LinkedList<Connection>();
+
+    @Test
+    public void assertloadAllTableNamesForOracle() throws SQLException {
+        LiteDatasource dataSource = new LiteDatasource();
+        SchemaMetaDataLoader.load(dataSource,5,"Oracle");
+    }
+
+    class LiteDatasource implements DataSource{
+        private String url = "jdbc:oracle:thin:@192.168.1.1:1521/orcl";
+        private String user = "test";
+        private String password = "111111";
+        public LiteDatasource(){
+            try {
+                for (int i = 0; i < SchemaMetaDataLoaderTest.maxIdleCount; i++) {
+                    Class.forName("oracle.jdbc.OracleDriver");
```

Comment:
What is the  jdbc driver ?

Reply:
I used externl lib to run this file.
Do you have another way to do?

Reply:
test ci will not pass, ,maybe you can mock oracle datasource to complete this PR.

Reply:
I need to connect to real oracle database. Is it work to mock oracle database?

Reply:
All of our database tests are mocks， If  a real database, how can someone else's environment have one？

Reply:
but the bug will not occur when mocking db nor an oracle db which just have one app user, it will just happen when an oracle db has multi users. Without real db, how can i prove i have fixed it.

Reply:
but the bug will not occur when mocking db nor an oracle db which just have one app user, it will just happen when an oracle db has multi users. Without real db, how can i prove i have fixed it.